File: reto_11.py
#Escribe un programa que reciba una lista de números (separados por comas) y luego: Muestre todos los números pares y Muestre todos los números impares
x = input("Escribe una lista de numeros separados por comas: ")
lista=x.split(",")
lista_pares = []
lista_impares= []
for item in lista:
    if int(item) % 2 == 0:
        lista_pares.append(int(item))
    else:
        lista_impares.append(int(item))
pares = ",".join(map(str, lista_pares))
impares = ",".join(map(str, lista_impares))
print(f"Los numeros pares son {pares}")
print(f"Los numeros impares son {impares}")

File: reto_11.js
//Escribe un programa que reciba una lista de números (separados por comas) y luego: Muestre todos los números pares y Muestre todos los números impares
const readline = require("readline");
const rl = readline.createInterface({
  input: process.stdin,  
  output: process.stdout 
});
rl.question("Ingresa números separados por comas: ", (input) => {
 let lista = input.split(",")
 let lista_pares = lista.filter(num => num % 2 === 0)
 let lista_impares = lista.filter(num => num % 2 !== 0)

 let pares = lista_pares.join(",")
 let impares = lista_impares.join(",")

 console.log(`los numeros pares son: ${pares}`)
 console.log(`los numeros impares son: ${impares}`)
});
// chatgpt usa el commando push y un if parecido al de python